packages/block-library/src/heading/edit.js

```diff
@@ -10,33 +15,37 @@ import { __ } from '@wordpress/i18n';
 import { Fragment } from '@wordpress/element';
 import { PanelBody } from '@wordpress/components';
 import { createBlock } from '@wordpress/blocks';
+import { withSelect } from '@wordpress/data';
 import {
 	RichText,
 	BlockControls,
 	InspectorControls,
 	AlignmentToolbar,
 } from '@wordpress/block-editor';
 
-export default function HeadingEdit( {
+function HeadingEdit( {
 	attributes,
 	setAttributes,
 	mergeBlocks,
 	insertBlocksAfter,
 	onReplace,
 	className,
+	levelChoices,
 } ) {
 	const { align, content, level, placeholder } = attributes;
 	const tagName = 'h' + level;
-
+	const BlockControlsLevelsRange = ( levelChoices.length <= 3 ) ?
```

Comment:
This should be camel-cased with an initial lower-case letter.

https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#naming-conventions

Reply:
Fixed